proto/cosmos/tx/signing/signing.proto

```diff
@@ -20,3 +22,49 @@ enum SignMode {
     // Amino JSON and will be removed in the future
     SIGN_MODE_LEGACY_AMINO_JSON = 127;
 }
+
+// SignatureDescriptors wraps multiple SignatureDescriptor's.
+message SignatureDescriptors {
+    // signatures are the signature descriptors
+    repeated SignatureDescriptor signatures = 1;
+}
+
+// SignatureDescriptor is a convenience type which represents the full data for a
+// signature including the public key of the signer, signing modes and the signature
+// itself. It is primarily used for coordinating signatures between clients.
+message SignatureDescriptor {
+    // public_key is the public key of the signer
+    cosmos.crypto.PublicKey public_key = 1;
+
+    Data data = 2;
+
+    // Data represents signature data
+    message Data {
+        // sum is the oneof that specifies whether this represents single or multi-signature data
+        oneof sum {
```

Comment:
We already have `message ModeInfo`, and if I understand this, this is ModeInfo+Signature. I'm still not 100% aware of protobuf extending types and such, but is there a way to use ModeInfo?

Reply:
Not without adding a field on ModeInfo that will be unused in transactions